core/src/main/java/com/google/common/truth/AbstractVerb.java

```diff
@@ -99,29 +117,40 @@ public S that(T target) {
     }
   }
 
-  protected static class MessagePrependingFailureStrategy extends FailureStrategy {
+  /**
+   * A {@link FailureStrategy} which delegates to an original strategy, and overrides the message
+   * when the various {@code fail()} methods are called, prepending the supplied formatted string
+   * to the Subject-supplied error message.
+   */
+  public static class DelegatingFailureStrategy extends FailureStrategy {
```

Comment:
private? This type is uninstantiable anywhere but here, isn't exposed as a type in the public API, and has no usable public API itself (at least as far as I can tell).